options/locale/locale_en-US.ini

```
@@ -946,6 +946,7 @@ clone_this_repo = Clone this repository
 create_new_repo_command = Creating a new repository on the command line
 push_exist_repo = Pushing an existing repository from the command line
 empty_message = This repository does not contain any content.
+broken_message = `The git data underlying this repository cannot be read. Contact the administrator of this instance or delete this repository.`
```

Are these ticks needed? If so LGTM

---

```suggestion
broken_message = The git data underlying this repository cannot be read. Contact the administrator of this instance or delete this repository.
```

Sorry I put them in because I was originally going to include something else but decided against it.